core/src/main/java/com/radixdlt/p2p/discovery/PeerDiscovery.java

```diff
@@ -136,6 +136,12 @@ public EventProcessor<DiscoverPeers> discoverPeersEventProcessor() {
       final var seedNodes = seedNodesConfigParser.getResolvedSeedNodes();
       this.addressBook.addUncheckedPeers(seedNodes);
 
+      if (seedNodes.isEmpty() && this.addressBook.knownPeers().isEmpty()) {
+        log.warn(
+            "Warning! No valid seed nodes have been configured for this node."
```

Comment:
"... and no peers in address book" ?
(or the `if` should only check `seedNodes`?)

Reply:
I did that on purpose :)
- if there are no seed nodes, but some peers have been added to addr book (e.g. other peers connected to us and address-book-synced), then it's fine - no need to log
- if there are no peers in the address book (yet) BUT there are some valid seed nodes - it's fine too, no need to log 
- if there are no peers in the address book AND none were configured - it's a warning, but the root cause is missing seed nodes (so no need to include "... and no peers in address book")